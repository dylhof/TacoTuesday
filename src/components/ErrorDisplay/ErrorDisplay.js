import React from 'react'

export const ErrorDisplay = () => {
  return(
    <h1> 404 not found</h1>
  )
}