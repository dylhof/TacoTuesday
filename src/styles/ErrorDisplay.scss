.notFoundMessage {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 20px;

  h2 {
    font-family: $title;
  }

  h3 {
    font-family: $text;
  }
}