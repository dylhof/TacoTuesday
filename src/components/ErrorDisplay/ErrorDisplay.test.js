import React from 'react';
import { shallow } from 'enzyme';
import {ErrorDisplay} from './ErrorDisplay';

describe('ErrorDisplay', () => {
  const mockPathName = '/yum-taco';
  let wrapper = shallow(<ErrorDisplay location={mockPathName}/>)

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});