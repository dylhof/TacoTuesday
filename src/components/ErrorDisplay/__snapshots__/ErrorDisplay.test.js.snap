// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDisplay should match the snapshot 1`] = `
<div
  className="notFoundMessage"
>
  <h2>
     404 Taco not found
  </h2>
  <h3>
     The path name:
  </h3>
  <h3>
    http://localhost:/3000
  </h3>
  <h3>
    does not contain any tacos
  </h3>
  <Link
    replace={false}
    to="/"
  >
    <WithStyles(Button)>
      Return To Tacos
    </WithStyles(Button)>
  </Link>
</div>
`;
